src/time.cr

```diff
@@ -503,20 +507,23 @@ struct Time
     utc(seconds: seconds, nanoseconds: nanoseconds.to_i)
   end
 
-  # Create a new `Time` instance that preserves the same wall clock reading than
-  # *time*  but in a different *location*.
+  # Create a new `Time` instance that preserves the same local date-time
+  # representation (wall clock) as *time* but observed in a different *location*.
   #
   # ```
   # time = Time.now(Time::Location.new(-3600))
   # time2 = Time.from_wall_clock(time, Time::Location.new(14400))
```

Comment:
Maybe we can two real timezones like New York and Tokyo and then call the vars `new_york_time`, `tokyo_wall_clock_time` or so? (Admittedly not great names yet). `time`, `time2` gets confusing fast here :/